settings.gradle

```diff
@@ -1,3 +1,5 @@
 include ':owncloud-android-library'
 include ':'
 include ':oc_jb_workaround'
+include ':touchImageView'
```

Comment:
Are you using this library? It was deleted when GIF support was implemented [here](https://github.com/owncloud/android/pull/2123/files#diff-c197962302397baf3a4cc36463dce5eaL43)